core/build.sbt

```diff
@@ -1,15 +1,10 @@
-import verizon.build.ScalazPlugin._
-
 resolvers ++= Seq(
-  "scalaz.bintray" at "http://dl.bintray.com/scalaz/releases",
   "oncue.bintray" at "http://dl.bintray.com/oncue/releases"
 )
 
 libraryDependencies ++= {
   Seq(
-    "org.scalaz.stream" %% "scalaz-stream" % scalazCrossVersioners.`scalaz-stream-0.8`(scalazVersion.value)("0.8.6"),
-    "io.verizon.ermine" %% "parser" % scalazCrossVersioners.default(scalazVersion.value)("0.4.7")
+    "co.fs2"            %% "fs2-core" % "0.10.0-M8",
```

Comment:
There's an M9 now, though it doesn't matter much because we'll release on 0.10.0.